ggml/src/ggml-cuda/cumsum.cu

```diff
@@ -61,14 +61,16 @@ static __global__ void cumsum_cub_kernel(
 
```

Comment:
Is the syncthreads at line 60 necessary? It seems like either that one or the one at line 78 isn't necessary.